pylsp_mypy/plugin.py

```diff
@@ -278,8 +278,10 @@ def init(workspace: str) -> Dict[str, str]:
 
     mypyConfigFile = findConfigFile(workspace, ["mypy.ini", ".mypy.ini"])
     mypyConfigFileMap[workspace] = mypyConfigFile
+    if mypyConfigFileMap[workspace] is None:
+        mypyConfigFileMap[workspace] = defaultMypyConfigFile()
```

Comment:
If no workspace config file (such as a local `mypy.ini` file) is found, this change will cause e.g. the flag `--config-file ~/.mypy.ini` to be passed to mypy.

I think this is sub-optimal, as `mypy` would automatically discover the `~/.mypy.ini` file and does not need to be told about `~/.mypy.ini` via a `--config-file` flag.